### PR TITLE
Sprites changes for old donator kit

### PR DESCRIPTION
**Pull request by SASoperative**
# About the pull request

Added sprites for the Juggernaut Donator kit. 

# Explain why it's good for the game

Replaces the old donator set which was using the vanilla syndicate hardsuit from SS13.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

(https://media.discordapp.net/attachments/1155292420343345182/1158740768677175396/jbTCHQLNCAANgNFAq26gCqW1JQoiUKTlCe6Fw6MLaXkbaVde0BxSxRAQAeyREQFokxREQARQEEypikNABCACiPOu7WoWAWzX9ot68lkQwKIQ02KFgBAQAitBQASwEkPqMYSAEBACXgREAF7EJC8EhIAQWAkCIoCVGFKPIQSEgBDwIiAC8CImeSEgBITAShAQAazEkHoMISAEhIAXARGAFzHJCwEhIARWgoAIYCWG1GMIASEgBLwIiAC8iEleCAgBIbASBP4POBL9vytFWmQAAAAASUVORK5CYII.png?ex=652e7c6f&is=651c076f&hm=12a12a220c0c86891b429785dacdfc940dde3ac22dddd9f346ecf3b05be15459&)

</details>

# Changelog
:cl:
add: Added donator item and sprites
/:cl:

### CODE REVIEW

File: code/modules/cm_marines/Donator_Items.dm [hunk 1]
```diff
@@ -1366,6 +1362,12 @@
 	icon_state = "securitypack"
 	item_state = "securitypack"
 
+/obj/item/storage/backpack/marine/satchel/fluff/sas_juggernaut //CKEY=sasoperative (UNIQUE)
+	name = "Tactical Radiopack"
```
Comment (Zonespace27) [inline code/modules/cm_marines/Donator_Items.dm:1366]: Names are lower case unless it's a proper noun, which this isn't

File: code/modules/cm_marines/Donator_Items.dm [hunk 2]
```diff
@@ -590,7 +593,7 @@
 
 /obj/item/clothing/head/helmet/marine/fluff/sas_juggernaut //CKEY=sasoperative (UNIQUE)
 	name = "Juggernaut Helmet"
```
Comment (Zonespace27) [inline code/modules/cm_marines/Donator_Items.dm:595]: Ditto

File: code/modules/cm_marines/Donator_Items.dm [hunk 3]
```diff
@@ -213,8 +216,8 @@
 /obj/item/clothing/suit/storage/marine/fluff/sas_juggernaut //CKEY=sasoperative (UNIQUE)
 	name = "Juggernaut Armor"
```
Comment (Zonespace27) [inline code/modules/cm_marines/Donator_Items.dm:217]: Ditto

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/cm_marines/Donator_Items.dm (hunk 1 above)
Comment (SASoperative) [reply code/modules/cm_marines/Donator_Items.dm]: Understood, changes made!

**Review by Zonespace27 — APPROVED** [review]

**Review by nauticall — APPROVED** [review]
sprites 👍

**Pull request merged**